engines/sci/engine/script_patches.cpp

```diff
@@ -18922,6 +18939,8 @@ void ScriptPatcher::processScript(uint16 scriptNr, SciSpan<byte> scriptData) {
 				// Enable subtitle compatibility if a sync resource is present
 				if (g_sci->getResMan()->testResource(ResourceId(kResourceTypeSync, 10))) {
 					enablePatch(signatureTable, "subtitle patch compatibility");
+				} else {
+					suggestDownloadGK2SubTitlesPatch();
```

Comment:
This should not be enforced for all users. The dialog should only be shown if the user has chosen to display "speech and subtitles" or "subtitles" in ScummVM's audio options

Reply:
OK.
Fixed that.
But before committing and pushing - a question.

Currently, the "edit game" menu for GK2 has "speech and subtitles"/"subtitles"/"speech" choosing disabled (probably because in the past it wasn't relevant).
I have changed that in my global "options" menu to verify my fix.

I'd like to enable that option in the GK2 "edit game" menu.

Is it better to be done on a separate PR?
Or in this PR as a separate commit?
Or even do it all on one commit?

Reply:
You need to check the "Override global audio settings" checkbox in order to override the defaults and change the audio settings for an individual game

Reply:
yeah, I know...

I already fixed that.
It was because of GUIO_NOSUBTITLES in GK2 detection table.
I've removed it, and it's fine now.

The question is technical-aesthetical, whether it's preffered to have the GUIO_NOSUBTITLES removal as a different commit, or even as a different PR, or send now one commit with everything.

Reply:
It would be best to end up with two commits, one in the common code, and one in the SCI engine